dashboard/config/environments/adhoc.rb

```diff
@@ -39,9 +39,6 @@
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
-  # Whether or not to display pretty apps (formerly called blockly).
-  config.pretty_apps = false
```

Comment:
is this change to adhoc config intended to be committed? it seems like it should go in a separate PR.